core/lib/mobility/plugins/store_based_fallbacks.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'concurrent'
+require 'mobility'
+
+module Mobility
+  @store_based_fallbacks = Concurrent::ThreadLocalVar.new(I18n::Locale::Fallbacks.new)
+
+  class << self
+    def store_based_fallbacks
+      @store_based_fallbacks.value
+    end
+
+    def store_based_fallbacks=(value)
+      @store_based_fallbacks.value = value
+    end
+  end
+
+  module Plugins
+    module StoreBasedFallbacks
+      extend ::Mobility::Plugin
+
+      default true
+      requires :backend, include: :before
+
+      # Applies fallbacks plugin to attributes. Completely disables fallbacks
+      # on model if option is +false+.
+      included_hook do |_, backend_class|
+        unless options[:fallbacks] == false
```

Comment:
It would be, but I'm not sure about the interface here :) After this change it will behave differently for a falsey value (e.g. nil), and I think Mobility's default fallbacks plugin would work if that option is not set. Only setting it explicitly to false disabled it.